algorithms/dqn/agent.py

```diff
@@ -153,30 +153,29 @@ def _get_dqn_loss(
     ) -> Tuple[torch.Tensor, torch.Tensor]:
         """Return element-wise dqn loss and Q-values."""
         states, actions, rewards, next_states, dones = experiences[:5]
-
-        q_values = self.dqn(states)
-        next_q_values = self.dqn(next_states)
-        next_target_q_values = self.dqn_target(next_states)
-
-        curr_q_value = q_values.gather(1, actions.long().unsqueeze(1))
-        next_q_value = next_target_q_values.gather(  # Double DQN
-            1, next_q_values.argmax(1).unsqueeze(1)
+        batch_size = self.hyper_params["BATCH_SIZE"]
```

Comment:
Create a key instead of using `str`?
```suggestion
        from params.keys import BATCH_SIZE
        batch_size = self.hyper_params[BATCH_SIZE]
```
```python
# params/keys.py
BATCH_SIZE = "BATCH_SIZE"
```

Reply:
I am looking for any way not to use strings as keys, like enum in c.
It would be better if I don't have to make a new .py to define keys.

Reply:
I made an issue: https://github.com/medipixel/rl_algorithms/issues/116